### ISSUE TITLE
Text For Links

### ISSUE THREAD

**Issue by dta121** [saylordotorg/moodle-theme_saylor#229]
Links should include text that is setting expectations to what page they are leading to. If empty links are used as layout wrapping
elements, a screen-reader only text or an "aria-label" attribute should be used for that description.

Successful elements:
<a class="nav-item nav-link" href="https://learn.saylor.org/?redirect=0">Courses</a>
<a class="nav-item nav-link" href="https://www.saylor.org/programs/">Programs</a>

Failed elements:
<a title="Summary" href="https://learn.saylor.org/course/info.php?id=360"><i class="icon fa fa-info fa-fw " title="Summary" aria-label="Summary"></i></a>
<a title="Summary" href="https://learn.saylor.org/course/info.php?id=349"><i class="icon fa fa-info fa-fw " title="Summary" aria-label="Summary"></i></a>

**Comment by jazinheira** [comment]
Is this mostly in the navbar, or other areas as well?
